src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import sys
+
+import requests
+
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def assert_can_communicate_with_server(cromshell_url):
+    """Check Connection with Cromwell Server"""
+
+    try:
+        curl_out = requests.get("{}/api/workflows/v1/backends".format(cromshell_url))
```

Comment:
Since we're not actually using `curl` it may be better not to use that in the variable name here.

Reply:
done